chains/solana/contracts/programs/ccip-router/src/instructions/v1/messages.rs

```diff
@@ -1,9 +1,17 @@
 use anchor_lang::prelude::*;
+use ethnum::U256;
 
-use crate::{TOKENPOOL_LOCK_OR_BURN_DISCRIMINATOR, TOKENPOOL_RELEASE_OR_MINT_DISCRIMINATOR};
+use crate::{
+    BillingTokenConfig, CcipRouterError, DestChain, Solana2AnyMessage, CHAIN_FAMILY_SELECTOR_EVM,
+    TOKENPOOL_LOCK_OR_BURN_DISCRIMINATOR, TOKENPOOL_RELEASE_OR_MINT_DISCRIMINATOR,
+};
 
 use super::pools::ToTxData;
 
+///////////////////
+// Pool Messages //
+///////////////////
```

Comment:
Nit: These kind of "category comments" always seem like a bit of a smell to me: I think they're hinting at the need of a module. I would use `pub mod pool {}` instead, so the namespace ends up looking like `messages::pool::`. Ditto for other categories.